### PR TITLE
SDI-2374: Support writing data points using UDP client

### PR DESCRIPTION
**Pull request by candysmurf**
Fixes: #106 
Summary of changes:
- added UDP support through the "scheme" config field
- updated README

How to verify it:
- manually run small and medium tests

Testing done:
- small and medium

Note that: large test will be fixed after this and Nan's PRs are merged.

### CODE REVIEW

**Comment by nanliu** [comment]
Ok, so you'll supply examples configs using UDP settings in the next PR along with large tests?

**Comment by candysmurf** [comment]
@nanliu, yes. Hopefully, after your large test is merged and based on that.

**Comment by nanliu** [comment]
Need to increment plugin version

File: influxdb/influxdb.go [hunk 1]
```diff
@@ -121,6 +126,11 @@ func (f *influxPublisher) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 	r9.Description = "Influxdb timestamp precision"
 	config.Add(r9)
 
+	r10, err := cpolicy.NewStringRule("scheme", true, HTTP)
+	handleErr(err)
+	r10.Description = "Influxdb communication protocol"
+	config.Add(r10)
+
```
Comment (nanliu) [inline influxdb/influxdb.go]: Should we remove r7, https?

File: influxdb/influxdb.go [hunk 2]
```diff
@@ -121,6 +126,11 @@ func (f *influxPublisher) GetConfigPolicy() (*cpolicy.ConfigPolicy, error) {
 	r9.Description = "Influxdb timestamp precision"
 	config.Add(r9)
 
+	r10, err := cpolicy.NewStringRule("scheme", true, HTTP)
```
Comment (kindermoumoute) [inline influxdb/influxdb.go]: The rule can't be required if there is a default value, it should be:
```Go
r10, err := cpolicy.NewStringRule("scheme", false, HTTP)
```

**Comment by candysmurf** [comment]
@nanliu & @kindermoumoute, changed. Please review it again. thanks.

**Pull request merged**